app/restore_to_file.go

```diff
@@ -145,64 +146,64 @@ func CopyFile(backingFilepath, outputFile string) error {
 }
 
 func CleanupTempFiles(outputFile string, files ...string) error {
+	defer logrus.Debugf("Cleanup Temp Files %v", files)
 	outputFilePath, err := filepath.Abs(outputFile)
 	if err != nil {
-		logrus.Errorf("find absolute path for output file=%s failed: %v", outputFile, err)
-		return err
+		return errors.Wrapf(err, "Failed find absolute path for output file=%s", outputFile)
```

Comment:
Lower case for the error in the `fmt.Errorf()` or `errors.Wrapf()` (`failed` vs `Failed`)

Also, `failed to find`, not `failed find`.